dataprofiler/profilers/profile_builder.py

```diff
@@ -1438,6 +1438,9 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0, options=No
         self.correlation_matrix = None
         self.chi2_matrix = None
 
+        # capitalone/synthetic-data specific metrics
+        self._null_replication_metrics = dict()
```

Comment:
instantiating class variables with `None` is best practice. Interesting we do it on line 1435. 

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
we do it on line 1438 and 1439